practice-two/src/pages/Analytics/Analytics.jsx

```diff
@@ -126,6 +142,7 @@ const Analytics = () => {
       {selectedCustomer && isShowProfileInfo && (
         <ProfileInfo selectedCustomer={selectedCustomer} />
       )}
+      {isShowForm && <FormValidation handleShowForm={() => setIsShowForm(false)} />}
```

Comment:
please consider handleShowForm of line 145 and line 41

Reply:
![image](https://github.com/datdragon777/React-Training/assets/74806826/b7f1716f-11db-43e6-a2bb-0bcde59afb9d)
![image](https://github.com/datdragon777/React-Training/assets/74806826/02a64993-58d2-402a-89a5-4b2a25a5de94)